### PR TITLE
Major reworking of the documentation

### LINKED ISSUE #1029: [FEAT]: Update the user documentation
### Is your feature request related to a problem? Please describe.

The user documentation is outdated.

### Describe the solution you'd like

Update the user documentation.

Because of new features, restructure where beneficial.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by ronzulu**
Fixes #1029,  #876

Available for reviewing at:
https://rjz-osr.s3.ap-southeast-2.amazonaws.com/site/en/index.html

### CODE REVIEW

File: docs/docs/en/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+!!! note ""
+
+    1. Display the [Note Review Queue](notes.md#note-review-queue) <br/>
+    2. Note review queue<br/>
+    3. Display the Obsidian command dialog to access the plugin [commands](plugin-commands.md)<br/>
+    4. `Flashcard Review Icon` Select a flashcard [deck](flashcards/reviewing.md#deck-selection) to [review](flashcards/reviewing.md#reviewing) <br/>
+    5. Identify that flashcards within this note are in the `#flashcards/science/physics` [deck](flashcards/decks.md#using-obsidian-tags)<br/>
+    6. A [single line question](flashcards/qanda-cards.md#single-line-basic) (identified by the `::` separating the question and answer)<br/>
+    7. The plugin stores scheduling info within this [HTML comment](data-storage.md#individual-markdown-files) <br/>
+    8. `Spaced Repetition Status Area` The number of notes and flashcards currently due for review. Click to SOMETHING.
```
Comment (st3v3nmw) [inline docs/docs/en/index.md:53]: > Click to SOMETHING.

This seems incomplete.

**Review by st3v3nmw — APPROVED** [review]
Thanks! This is an awesome change... There's one incomplete statement but otherwise this LGTM.

File: docs/docs/en/index.md (hunk 1 above)
Comment (ronzulu) [reply docs/docs/en/index.md]: Glad you like the changes :-)
And thanks for picking up the mistake

**Comment by ronzulu** [comment]
Hi @st3v3nmw thanks for merging.

I fixed some references just now, could you please merge again.

Details in https://github.com/st3v3nmw/obsidian-spaced-repetition/pull/1048#issuecomment-2290450443

**Comment by st3v3nmw** [comment]
> could you please merge again

I think we'd need a new pull request 😃

**Comment by ronzulu** [comment]
Good point, see https://github.com/st3v3nmw/obsidian-spaced-repetition/pull/1051

**Pull request merged**